doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -907,3 +907,15 @@
     // iox::string to std::string
     std::string myConvertedIoxString = iox::cxx::into<std::string>(myIoxString);
     ```
+
+44. Move and rename `DeadlineTimer`
```

Comment:
Both `adaptive_wait` and `units::Duration` are labeled as `internal` and hence not mentioned in the release note.

Reply:
@mossmaurice since `unints::Duration` might be used quite heavily despite being labeled `internal` I think it would nevertheless be a good idea to also mention that class.

Reply:
@elBoberido Good point, added.